src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -40,19 +40,6 @@ public MobileAdsBaseObject(WebDriver driver, String page) {
     PageObjectLogging.log("", "Page screenshot", true, driver);
   }
 
-  @Override
-  protected void setWindowSizeAndroid() {
-    try {
-      driver.manage().window().setSize(new Dimension(360, 640));
-    } catch (WebDriverException ex) {
-      PageObjectLogging.log(
-          "ResizeWindowForMobile",
-          "Resize window method not available - possibly running on real device",
-          true
-      );
-    }
-  }
-
   public void verifyMobileTopLeaderboard() {
     removeElementIfPresent(SMART_BANNER_SELECTOR); // Only works for WikiaMobile
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)